_posts/2019-01-24-PolicyGradients.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title:  "Policy Gradients"
+date:   2019-01-24 11:42:04 +0530
+categories: Reinforcement Learning
+author: Madhuparna Bhowmik
+---
+
+In this article, we will learn about Policy Gradients and implement it in Pytorch.
+
+## Introduction
+
+Policy Gradients method is a model-free Reinforcement Learning algorithm that directly approximates the Policy. 
+
+In the previous blog posts, we saw Q-learning based algorithms like DQN and DRQNs where given a state we were finding the Q-values of the possible actions where the Q-values are the expected reward we can get from that state if that action is selected. And we were using an epsilon-greedy strategy to select the action. However, in Policy Gradients method instead of approximating the action value function, we are directly learning the optimal policy. 
```

Comment:
"where the Q-values are the expected return for the episode we can get from that state if that action is selected"